spec/spec_helper_acceptance.rb

```diff
@@ -18,7 +18,7 @@ def create_keys_for_test(host)
     path = '${PATH}'
     path = "/opt/csw/bin:#{path}" # Need ruby's path on solaris 10 (foss)
     if host.is_pe?
-      path = "#{host['puppetbindir']}:#{path}" # But try PE's ruby first
+      path = "#{host['puppetbindir']}:/opt/puppetlabs/puppet/bin:#{path}" # But try PE's ruby first
     end
```

Comment:
I don't know if aio agents will respond to is_pe? since there's only distinction on the master/pdb/console servers.